tools/webdriver/webdriver/bidi/modules/permissions.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, Optional, Mapping, MutableMapping
+
+from ._module import BidiModule, command
+
+
+class Permissions(BidiModule):
+    @command
+    def end(self) -> Mapping[str, Any]:
+        return {}
+
+    @end.result
+    async def _end(self, result: Mapping[str, Any]) -> Any:
+        if self.session.transport:
+            await self.session.transport.wait_closed()
+
+        return result
```

Comment:
I'm not sure why this is required.

Reply:
Removed. I think it was from another module.